src/instance-group-manager.js

```diff
@@ -153,7 +155,154 @@ class InstanceGroupManager extends common.ServiceObject {
+      {
+        method: 'POST',
+        uri: '/deleteInstances',
+        json: {
+          instances: arrify(vms).map(function(vm) {
+            return vm.url;
+          }),
+        },
+      },
+      function(err, resp) {
+        if (err) {
+          callback(err, null, resp);
+          return;
+        }
+        const operation = self.zone.operation(resp.name);
+        operation.metadata = resp;
+        callback(null, operation, resp);
+      }
+    );
+  }
+  /**
+   * Get a list of managed VM instances in this instance group manager.
+   *
+   * @see [InstanceGroupManagers: listManagedInstances API Documentation]{@link https://cloud.google.com/compute/docs/reference/v1/instanceGroupManagers/listManagedInstances}
+   *
+   * @param {object=} options - Instance search options.
+   * @param {boolean} options.autoPaginate - Have pagination handled
+   *     automatically. Default: true.
+   * @param {string} options.filter - Search filter in the format of
+   *     `{name} {comparison} {filterString}`.
+   *     - **`name`**: the name of the field to compare
+   *     - **`comparison`**: the comparison operator, `eq` (equal) or `ne`
+   *       (not equal)
+   *     - **`filterString`**: the string to filter to. For string fields, this
+   *       can be a regular expression.
+   * @param {number} options.maxApiCalls - Maximum number of API calls to make.
+   * @param {number} options.maxResults - Maximum number of VMs to return.
+   * @param {string} options.pageToken - A previously-returned page token
+   *     representing part of the larger set of results to view.
+   * @param {function} callback - The callback function.
+   * @param {?error} callback.err - An error returned while making this request.
+   * @param {VM[]} callback.vms - VM objects from this instance
+   *     group.
+   * @param {object} callback.apiResponse - The full API response.
+   *
+   * @example
+   * const Compute = require('@google-cloud/compute');
+   * const compute = new Compute();
+   * const zone = compute.zone('us-central1-a');
+   * const instanceGroupManager = zone.instanceGroupManager('web-servers');
+   *
+   * instanceGroupManager.listManagedInstances(function(err, vms) {
+   *   // `vms` is an array of `VM` objects.
+   * });
+   *
+   * //-
+   * // To control how many API requests are made and page through the results
+   * // manually, set `autoPaginate` to `false`.
+   * //-
+   * function callback(err, vms, nextQuery, apiResponse) {
+   *   if (nextQuery) {
+   *     // More results exist.
+   *     instanceGroupManager.listManagedInstances(nextQuery, callback);
+   *   }
+   * }
+   *
+   * instanceGroupManager.listManagedInstances({
+   *   autoPaginate: false
+   * }, callback);
+   *
+   * //-
+   * // If the callback is omitted, we'll return a Promise.
+   * //-
+   * instanceGroupManager.listManagedInstances().then(function(data) {
+   *   const vms = data[0];
+   * });
+   */
+  listManagedInstances(options, callback) {
```

Comment:
"getManagedInstances", please!

Reply:
Fixed in f9c0601a7aabd04c29649bfa167e7cd75a48b48a.